apis/nucleus/src/hooks/useAppSelections.js

```diff
@@ -59,7 +59,6 @@ function createAppSelections({ app, currentSelectionsLayout, navState }) {
 
   /**
    * @class
-   * @hideconstructor
```

Comment:
why are you removing @hideconstructor? (here and in other places)

Reply:
The spec is invalid with it, not sure if the schema is correct or not, but removing it won't affect the docs.